Vision/SplitImage/SplitImage.cs

```diff
@@ -47,16 +55,36 @@ static SplitImage()
 
             WebApiSkillResponse response = WebApiSkillHelpers.ProcessRequestRecords(skillName, requestRecords,
                 (inRecord, outRecord) => {
-                    var imageUrl = inRecord.Data["imageLocation"] as string; // no url parameters
-                    var sasToken = inRecord.Data["sasToken"] as string; // includes ? to start query params
+                    object imageUrlObject = null;
+                    object sasTokenObject = null;
+                    inRecord.Data.TryGetValue("imageLocation", out imageUrlObject);
```

Comment:
I recently noticed you can write this `var imageUrl = (inRecord.Data.TryGetValue("imageLocation", out object imageUrlObject) ? imageUrlObject : null) as string;`, which I prefer, but it's a question of taste I suppose. Just a suggestion.

Should use `var` though because of the `as`.